JobConfig/primary/IPAMuminusMichel.fcl

```diff
@@ -6,16 +6,57 @@
 #
 #include "Production/JobConfig/primary/IPAStopParticle.fcl"
 
-physics.filters.PrimaryFilter.MinimumPartMom : 30.0 // MeV/c TODO - tweak this parameter to get more DetSteps
+physics.filters.PrimaryFilter.MinimumPartMom : 1.0
 physics.producers.FindMCPrimary.PrimaryProcess : "mu2eMuonDecayAtRest"
 
-physics.producers.generate : {
-  module_type : Pileup
-  verbosity : 0
-  inputSimParticles: IPAMuminusStopResampler
-  captureProducts : []
-  decayProducts : [ @local::Pileup.dioGenTool ]
-  stoppingTargetMaterial : "IPA"
+CompositeGenerator: {
```

Comment:
The base definition of a module configuration belongs in Offline.  In Production, reference it with "@table::" here and override anything that needs to be overridden.  Just as you do for GenFilter, below.

Reply:
All of the analogous primary generators are configured directly as definition of `physics.producers.generate`. Here I defined it as a standalone object (? I'm not sure the right terminology here) and then defined `physics.producers.generate` as that object, which I guess does violate the rule you cite. I've now switched to directly define `physics.producers.generate`, consistent with the other primaries.

Reply:
Ok.  I see that they all do it that way.  Another thing to fix in a later PR.  The reason we would like the base fcl definition defined in ../fcl/prolog.fcl relative to the module source code is to minimize multiple points of maintenance.